### PR TITLE
python-six,et_xmlfile, django: bump package versions

### PR DESCRIPTION
**Pull request by commodo**
Maintainer: me (for all 3), @jefferyto (for python-six), @peter-stadler (for django), @cotequeiroz (for et_xmlfile)
Compile tested: x86 https://github.com/openwrt/openwrt/commit/97e820c6d61d174da1091a692546e979a5a45e42
Run tested: x86 https://github.com/openwrt/openwrt/commit/97e820c6d61d174da1091a692546e979a5a45e42

Signed-off-by: Alexandru Ardelean <ardeleanalex@gmail.com>

### CODE REVIEW

File: lang/python/python-et_xmlfile/Makefile [hunk 1]
```diff
@@ -6,14 +6,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-et_xmlfile
-PKG_VERSION:=1.0.1
-PKG_RELEASE:=2
+PKG_VERSION:=1.1.0
+PKG_RELEASE:=1
 
 PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>, Eneas U de Queiroz <cote2004-github@yahoo.com>
```
Comment (cotequeiroz) [inline lang/python/python-et_xmlfile/Makefile:12]:
```suggestion
PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>, Eneas U de Queiroz <cotequeiroz@gmail.com>
```

**Review by cotequeiroz — APPROVED** [review]
@commodo, can you please update my email address while at it?
Nonetheless, LGTM

File: lang/python/python-et_xmlfile/Makefile (hunk 1 above)
Comment (peter-stadler) [reply lang/python/python-et_xmlfile/Makefile:12]: LGTM, too.

**Comment by peter-stadler** [comment]
LGTM, too :-)

File: lang/python/python-et_xmlfile/Makefile (hunk 1 above)
Comment (commodo) [reply lang/python/python-et_xmlfile/Makefile:12]: sure

**Comment by commodo** [comment]
Changelog v1 -> v2:
* updated Eneas' email

**Comment by commodo** [comment]
Changelog v2 -> v3:
* rebased on newer master
* bumped Django to 3.2.3 ; another version appeared since this PR was opened;

**Pull request merged**